ios/Exponent/Supporting/Info.plist

```diff
@@ -131,6 +131,7 @@
 		<string>audio</string>
 		<string>fetch</string>
 		<string>location</string>
+		<string>remote-notification</string>
```

Comment:
cc @brentvatne I think you had some hesitation around including a background mode in Expo Go, but looks like `location`, `fetch`, and `audio` are already included anyways so I don't think this will impact app review, but lmk your thoughts

Reply:
nvm- this is now removed as part of our fastlane build script